code/code/misc/being.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef __BEING_H
+#define __BEING_H
```

Comment:
Curious why you're changing this?

Reply:
Oh yeah, meant to make a note of that and forgot. My linter was giving warnings about redefinitions of various classes and stuff in being.h, and using #ifndef instead fixed them. From researching it, apparently `#pragma once` isn't as widely supported and #ifndef is more reliable. But if you'd rather use #pragma I'm happy to change it back. I noticed that #ifndef is used pretty widely throughout the rest of the codebase, though.

Reply:
I really was under impression that everything suppors `#pragma once`. If your linter doesn't, well, maybe get a better linter?

Reply:
Tho I'm not really insisting :) Either is fine by me.

Reply:
Took your advice and fixed my linter. Reverting this back to using `#pragma once` in https://github.com/sneezymud/sneezymud/pull/387/commits/42a634cd73fbed333a5f5fb8f4a136a35c325ad7.